tests/RegexTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\MakerBundle\Test\MakerTestEnvironment;
+
+class RegexTest extends TestCase
```

Comment:
Calling this `MakeTestEnvironmentTest` was not compatible with PHPUnit `--filter` (test wouldn't run) and I'm not sure it was running w/ just `simple-phpunit` && no flags either...